controller/contactUs.js

```diff
@@ -1,5 +1,27 @@
 const Contacts = require("../models/Contact")
 
+// Intern Validation rules
+const contactValidationRules = () => [
+  body('name').isString(),
+  body('email').isEmail(),
+  body('subjec').isString(),
+  body('message').isString(),
+];
+
+
+// get all contacts
+const getAllContacts = (req, res) => {
```

Comment:
this function is not suppose to be in here... check my admin view in controller/view/index

Reply:
should it be removed?